server/lib/configuration.js

```diff
@@ -123,15 +123,18 @@ const conf = module.exports = convict({
   experiments: {
     dir: {
       default: path.resolve(__dirname, '..', '..', 'experiments'),
-      doc: 'path to where experiments are stored'
+      doc: 'path to where experiments are stored',
+      env: 'EXPERIMENTS_STORAGE_DIR'
     },
     git: {
       default: 'github:mozilla/fxa-content-experiments#dev',
-      doc: 'git url for experiments repo. set to empty to not pull'
+      doc: 'git url for experiments repo. set to empty to not pull',
+      env: 'EXPERIMENTS_GIT_REPO'
     },
     watch: {
       default: false,
-      doc: 'poll the experiments git repo for changes'
+      doc: 'poll the experiments git repo for changes',
+      env: 'EXPERIMENTS_POLL'
```

Comment:
Shouldn't we also specify a `format` for this one to ensure it coerces to `Boolean` correctly?

Also, the current `doc` and `env` don't make clear that we're looking for an on/off value here, only the `default` does. Should we update them to make the booleanness explicit, `EXPERIMENTS_ENABLE_POLLING`?

(the current name/description could equally apply to, say, milliseconds polling interval)